transpose.json

```diff
@@ -0,0 +1,206 @@
+{
+  "#": [
+      "JSON doesn't allow for multi-line strings, so all multi-line input is ",
+      "presented here as arrays of strings. It's up to the test generator to join the ", 
+      "lines together with line breaks."
+  ],
+  "cases": [
+    {
+      "description": "empty string",
+      "input": "",
+      "expected": ""
+    },
+    {
+      "description": "two characters",
+      "input": "A1",
+      "expected": [ 
+          "A",
+          "1"
+        ]
+    },
+    {
+      "description": "simple",
+      "input": [ 
+          "ABC",
+          "123"
+        ],
+      "expected": [ 
+          "A1",
+          "B2",
+          "C3"
+        ]
+    },
+    {
+      "description": "single line",
+      "input": "Single line.",
+      "expected": [ 
+          "S",
+          "i",
+          "n",
+          "g",
+          "l",
+          "e",
+          " ",
+          "l",
+          "i",
+          "n",
+          "e",
+          "."
+        ]
+    },
+    {
+      "description": "two lines",
+      "input": [ 
+          "The first line.",
+          "The second line."
+        ],
+      "expected": [ 
+          "TT",
+          "hh",
+          "ee",
+          "  ",
+          "fs",
+          "ie",
+          "rc",
+          "so",
+          "tn",
+          " d",
+          "l ",
+          "il",
+          "ni",
+          "en",
+          ".e",
+          "." 
```

Comment:
in the original text, the period belongs to the second line, so there should be a space before it in the transposed text